app/domain/labor-market/functions.server.ts

```diff
@@ -47,6 +47,20 @@ export async function upsertIndexedLaborMarket(address: EvmAddress, block?: numb
     serviceRequestRewardPools: [],
     createdAtBlockTimestamp: new Date(),
   };
+
+  //log this event in user activity collection
+  mongo.userActivity.insertOne({
+    groupType: "LaborMarket",
+    eventType: "LaborMarketConfigured",
+    actionName: "Create Marketplace",
+    userAddress: configuration.owner,
```

Comment:
Think that instead of having these event specific attributes at the top level we can nest them in an object called `config` or something. That way all the different event types can store arbitrary data.

Parsing that object will then be the tricky part since it will depend on the `eventType`, but certainly possible.